### DISCUSSION TITLE
Syncing between Plex and Jellyfin. Add another Plex server to sync

### DISCUSSION THREAD

**Discussion by charlieny100** [luigi311/JellyPlex-Watched#177] [Q&A]
I'm currently syncing between my Plex server and my Jellyfin server. I also have backup Plex server that I'd like to get updated watch status. I tried adding the second Plex server as "http://server1:32400,http://server2:32400" but it errored out. Do I need another container for the Plex to Plex sync instead of attempting to do it all in one container?

**Comment by luigi311** [comment]
that should work, did you also add the new token? What is it saying the in the error?

**Reply by charlieny100** [reply]
I saw in another post someone said to use the same token. Let me add the second servers token and test it. In the PLEX_TOKEN field is it “token1,token2”. I’m doing this on unraid.
